package.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "chalk-template",
```

Comment:
If you'll be publishing this to npm, this `name` field needs to be changed, unless you have access to the existing npm package named `chalk-template`:

https://www.npmjs.com/package/chalk-template

Reply:
Ah, looks like you do! Looking forward 😊

![image](https://user-images.githubusercontent.com/251288/139742615-e010f0b5-3b90-422e-9a35-be294cc34d5a.png)